packages/pages/src/dev/server/ssr/generateTestData.ts

```diff
@@ -175,3 +179,17 @@ const prepareJsonForCmd = (json: any) => {
   }
   return jsonString;
 };
+
+// There is a chance multiple documents will be returned consecutively, without proper containment. Look for them and parse as an array
```

Comment:
can you make a note that versions of the CLI <= XX return documents without the containment?

Also just an improvement for the future but pinning certain versions of pagesjs to the CLI will be pretty necessary. Unfortunately the CLI isn't semantically versioned yet.